bodhi/server/models.py

```diff
@@ -1756,13 +1764,15 @@ def set_request(self, db, action, username):
                         'stable repository.  It must first reach a karma of %s, consisting of %s '
                         'positive karma from proventesters, along with %d additional karma from '
                         'the community. Or, it must spend %s days in testing without any negative '
-                        'feedback')
+                        'feedback.')
                     stern_note = stern_note % (
                         config.get('critpath.min_karma'),
                         config.get('critpath.num_admin_approvals'),
                         (config.get('critpath.min_karma') -
                             config.get('critpath.num_admin_approvals')),
                         config.get('critpath.stable_after_days_without_negative_karma'))
+                    if config.get('greenwave_enabled'):
+                        stern_note += ' Or, it must pass gating.'
```

Comment:
Also, the CI failed because this line is missing test coverage. If you are using the Vagrant development environment, ```bdiff-cover``` can tell you if you have full coverage on your patch.